src/Integration/Integration.csproj

```diff
@@ -256,11 +257,17 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```

Comment:
Why have `app.config`s been added to some projects? I see it's for Json.NET, but are they needed?